src/Components/AdvancedSearch/AdvancedSearch.tsx

```diff
@@ -123,6 +132,23 @@ const AdvancedSearch: React.FC<IAdvancedSearchProps> = (props) => {
                             }
                         }}
                     />
+                    <div>
+                        <Stack
```

Comment:
the buttons should not float up. They should get pegged to the bottom. 
Look into whether we can use a ModalFooter (I think that's what it's called) and then Modal just takes care of the scrolling the content and all that jazz for us.
![image](https://user-images.githubusercontent.com/57726991/189770459-f7569ff6-5694-4c02-9892-03088b92ae91.png)